### DISCUSSION TITLE
No sound with DownloadHelper 8.2.1.1 for Chrome

### DISCUSSION THREAD

**Discussion by nicolas-cahorel** [aclap-dev/video-downloadhelper#238] [Help and Q&A]
### Your question

Hi ! 
I'm trying to download a replay from a news channel for a friend. The downloaded mp4 file doesn't contain an audio track.
hope someone can solve this issue.
Thanks

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: fr
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\nicol",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi,
Is it possible to have a public link of a video with this issue?

**Comment by nicolas-cahorel** [comment]
here is the URL : https://www.20minutes.tv/video/un-hopital-propose-des-seances-de-mediation-avec-des-anes/

**Comment by paulrouget** [comment]
Source audio format is buggy, it has an unspecified sample rate. I think I can work around that… investigating.

**Reply by nicolas-cahorel** [reply]
Thank you for your help ! I am looking forward for an improvement.

**Comment by paulrouget** [comment]
Alright. Got a fix. Hope to get that ready by version 8.2.1.9.

**Reply by GrampaWildWilly** [reply]
Confirming your observation that there's something wrong with the audio on this item:

[ffprobe.txt](https://github.com/aclap-dev/video-downloadhelper/files/14459014/ffprobe.txt)

**Reply by GrampaWildWilly** [reply]
Is your fix another source correction to ffmpeg?

**Comment by paulrouget** [comment]
> Is your fix another source correction to ffmpeg?

No. I just increased the sampling analyze duration.

**Reply by GrampaWildWilly** [reply]
Smacking my forehead.  The parameter name is in the ffprobe error message.  Actually, I experimented a little & I needed to supply both the parameters named in the error message.  So here's what I did:

[ffprobe.txt](https://github.com/aclap-dev/video-downloadhelper/files/14460723/ffprobe.txt)
[Émission français mp4 Log.txt](https://github.com/aclap-dev/video-downloadhelper/files/14460731/Emission.francais.mp4.Log.txt)

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/753ca4f3-026d-4c36-bb9b-5442e6bd8ca3)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/efdc20c0-722c-4fe8-b335-c34882315851)

Maybe I chose values for those parameters that are ridiculously large.  But I tried a couple of values I thought were more reasonable & they didn't work.  So this is what I went with.  They seem innocuous enough.  I assume this is what's going to be in the next CoApp.

I would like to say I enjoyed the clip but I generally can't understand the way people from France speak French.  My ear is more attuned to les montréalais, like Martin McGuire & Dany Dubé.  I would probably have to live en France for a few months before I would get used to the accent.  Oh well.

**Reply by GrampaWildWilly** [reply]
I wonder if it's worth the trouble to make those 2 parameters into VDH settings.  It's possible that different values might be necessary in different situations.  Or is there a large enough value that should work in most cases & you'll just code a larger value should experience show it needs to be larger?
